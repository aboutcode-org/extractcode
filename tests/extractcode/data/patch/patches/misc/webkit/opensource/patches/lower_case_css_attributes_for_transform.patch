--- a/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp.orig	2008-11-26 14:30:09.000000000 +0100
+++ b/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp	2009-08-24 15:52:19.000000000 +0200
@@ -3374,30 +3374,30 @@ public:
     , m_allowSingleArgument(false)
     , m_unit(CSSParser::FUnknown)
     {
-        if (fname == "scale(" || fname == "scaleX(" || fname == "scaleY(") {
+        if (fname == "scale(" || fname == "scalex(" || fname == "scaley(") {
             m_unit = CSSParser::FNumber;
             if (fname == "scale(")
                 m_type = CSSTransformValue::ScaleTransformOperation;
-            else if (fname == "scaleX(")
+            else if (fname == "scalex(")
                 m_type = CSSTransformValue::ScaleXTransformOperation;
             else
                 m_type = CSSTransformValue::ScaleYTransformOperation;
         } else if (fname == "rotate(") {
             m_type = CSSTransformValue::RotateTransformOperation;
             m_unit = CSSParser::FAngle;
-        } else if (fname == "skew(" || fname == "skewX(" || fname == "skewY(") {
+        } else if (fname == "skew(" || fname == "skewx(" || fname == "skewy(") {
             m_unit = CSSParser::FAngle;
             if (fname == "skew(")
                 m_type = CSSTransformValue::SkewTransformOperation;
-            else if (fname == "skewX(")
+            else if (fname == "skewx(")
                 m_type = CSSTransformValue::SkewXTransformOperation;
             else
                 m_type = CSSTransformValue::SkewYTransformOperation;
-        } else if (fname == "translate(" || fname == "translateX(" || fname == "translateY(") {
+        } else if (fname == "translate(" || fname == "translatex(" || fname == "translatey(") {
             m_unit = CSSParser::FLength | CSSParser::FPercent;
             if (fname == "translate(")
                 m_type = CSSTransformValue::TranslateTransformOperation;
-            else if (fname == "translateX(")
+            else if (fname == "translatex(")
                 m_type = CSSTransformValue::TranslateXTransformOperation;
             else
                 m_type = CSSTransformValue::TranslateYTransformOperation;
