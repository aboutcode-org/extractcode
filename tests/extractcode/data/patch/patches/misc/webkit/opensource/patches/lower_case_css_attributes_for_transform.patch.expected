[('a/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp.orig',
  'b/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp',
  '--- a/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp.orig\n+++ b/src/3rdParty/webkit/work/WebCore/css/CSSParser.cpp\n@@ -3374,30 +3374,30 @@ public:\n, m_allowSingleArgument(false)\n, m_unit(CSSParser::FUnknown)\n{\n-        if (fname == "scale(" || fname == "scaleX(" || fname == "scaleY(") {\n+        if (fname == "scale(" || fname == "scalex(" || fname == "scaley(") {\nm_unit = CSSParser::FNumber;\nif (fname == "scale(")\nm_type = CSSTransformValue::ScaleTransformOperation;\n-            else if (fname == "scaleX(")\n+            else if (fname == "scalex(")\nm_type = CSSTransformValue::ScaleXTransformOperation;\nelse\nm_type = CSSTransformValue::ScaleYTransformOperation;\n} else if (fname == "rotate(") {\nm_type = CSSTransformValue::RotateTransformOperation;\nm_unit = CSSParser::FAngle;\n-        } else if (fname == "skew(" || fname == "skewX(" || fname == "skewY(") {\n+        } else if (fname == "skew(" || fname == "skewx(" || fname == "skewy(") {\nm_unit = CSSParser::FAngle;\nif (fname == "skew(")\nm_type = CSSTransformValue::SkewTransformOperation;\n-            else if (fname == "skewX(")\n+            else if (fname == "skewx(")\nm_type = CSSTransformValue::SkewXTransformOperation;\nelse\nm_type = CSSTransformValue::SkewYTransformOperation;\n-        } else if (fname == "translate(" || fname == "translateX(" || fname == "translateY(") {\n+        } else if (fname == "translate(" || fname == "translatex(" || fname == "translatey(") {\nm_unit = CSSParser::FLength | CSSParser::FPercent;\nif (fname == "translate(")\nm_type = CSSTransformValue::TranslateTransformOperation;\n-            else if (fname == "translateX(")\n+            else if (fname == "translatex(")\nm_type = CSSTransformValue::TranslateXTransformOperation;\nelse\nm_type = CSSTransformValue::TranslateYTransformOperation;')]